.circleci/config.yml

```
@@ -526,6 +529,109 @@ jobs:
             cd priv/perf
             mix do credo, format --check-formatted --dry-run
 
+  test_docker_compose_reorg:
+    machine:
+      image: ubuntu-1604:201903-01
+    environment:
+      REORG: true
+    steps:
+      - checkout
+      - run:
+          name: Setup data dir
+          command: |
+            [ -d data1 ] || mkdir data1 && chmod 777 data1
+            [ -d data2 ] || mkdir data2 && chmod 777 data2
+            [ -d data ] || mkdir data && chmod 777 data
+      - run: make docker-child_chain
+      - run: make docker-watcher
+      - run: make docker-watcher_info
+      - run:
+          name: Start daemon services
+          command: |
+            cd priv/cabbage
+            make start_daemon_services_reorg || (START_RESULT=$?; docker-compose logs; exit $START_RESULT;)
+      - run:
+          name: Print docker states
+          command: |
+            docker image ls
+            docker-compose ps
+      - restore_cache:
+          key: v2-asdf-install
+      - run:
+          name: Install Erlang and Elixir
+          command: |
+            [ -d ~/.asdf-vm ] || git clone https://github.com/asdf-vm/asdf.git ~/.asdf-vm --branch v0.7.4
+            echo 'source ~/.asdf-vm/asdf.sh' >> $BASH_ENV
+            source $BASH_ENV
+            asdf plugin-add erlang || asdf plugin-update erlang
+            asdf plugin-add elixir || asdf plugin-update elixir
+            asdf install erlang 22.3
+            asdf install elixir 1.10.2
+            asdf global elixir 1.10.2
+            asdf global erlang 22.3
+          no_output_timeout: 2400
+      - save_cache:
+          key: v2-asdf-install
+          paths:
+            - ~/.asdf
+            - ~/.asdf-vm
+      - run: make install-hex-rebar
+      - run: sh .circleci/status.sh
+      - restore_cache:
+          key: v2-mix-specs-cache-{{ .Branch }}-{{ checksum "mix.lock" }}
+      - run:
+          name: Print reorg logs
+          command: docker-compose -f docker-compose-reorg.yml logs --follow | grep "reorg"
+          background: true
+      - run:
+          name: Run specs
+          command: |
+            cd priv/cabbage
+            make install
+            make generate_api_code
+            mix deps.get
+            mix test --exclude in_flight_exits --exclude invalid_standard_exits_tests --exclude transactions --exclude standard_exits_tests --exclude watcher_status --exclude watcher_info_api --exclude configuration_retrieval --exclude service_name --exclude fee_claiming --trace
```

why all the excludes? What is left for `mix test` to run? Could this just be something like

```suggestion
            mix test --only some_tag --trace
```

---

I added `reorg` tag